portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -21,23 +21,32 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.util.List;
+import java.util.ArrayList;
 import java.util.Arrays;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private List<String> comments = new ArrayList<String>();  // Stores all the comments for now
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-
-    List<String> planets = Arrays.asList("Mercury","Venus","Earth","Mars","Jupiter","Saturn","Uranus","Nepturne");
     response.setContentType("application/json;");
-    response.getWriter().println(toJson(planets));
+    response.getWriter().println(toJson(comments));
+  }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException{ 
+    comments.add(request.getParameter("comment"));
```

Comment:
When there is no comment at all, you may want to check against this case.
One good helper function from the example code:

```
private String getParameter(HttpServletRequest request, String name, String defaultValue) {
    String value = request.getParameter(name);
    if (value == null) {
      return defaultValue;
    }
    return value;
  }
```

Reply:
Oh, that is right. Thank you @qidichen .

I am thinking of taking care of this use case on the client side by restricting the form even trying to send a null value to the Servlet. By using the html `required` attribute as follow. 
```
<input required> </input>
```
**Advantage**:  The page won't have to reload .

Please let me know if there is anything I  am forgetting to consider with this approach.

Thanks again.